docs/quick-reference.adoc

```diff
@@ -1 +1,66 @@
 = Sylt Quick Reference
+
+Here you will find most things in Sylt in a concise format.
+
+.Variables
+[source, sylt]
+----
+constant :: 3.14
+variable := []
+two: int : 2
+text: str = "abc"
+----
+
+.If-else statements
+[source, sylt]
+----
+if 1 < 0 do
+    // ...
+else if "hello" == "" do
+    // ...
+else do
+    // ...
+end
+
+one :: 1 if true else 2
+two :: one + 1 if == 2 else -1 // <1>
```

Comment:
Better example? These don't give me a lot of information. Maybe an example function?

Reply:
I know the example does not show that much, but perhaps it will not be a problem when we can link to the more comprehensive guide.